networkx/algorithms/bipartite/generators.py

```diff
@@ -551,8 +551,8 @@ def gnmk_random_graph(n, m, k, seed=None, directed=False):
 
     Examples
     --------
-    from nx.algorithms import bipartite
-    G = bipartite.gnmk_random_graph(10,20,50)
+    >>> from networkx.algorithms import bipartite
+    >>> G = bipartite.gnmk_random_graph(10, 20, 50)
```

Comment:
Might as well take the opportunity to update the access pattern:

```suggestion
    >>> G = nx.bipartite.gnmk_random_graph(10, 20, 50)
```

Reply:
Good catch! Done. I was too distracted by the original typo `from nx.algorithms import bipartite`